qa/L0_trt_shape_tensors/trt_shape_tensor_test.py

```diff
@@ -204,13 +206,14 @@ def test_wrong_shape_values(self):
             iu.infer_shape_tensor(
                 self,
                 'plan',
-                8,
                 np.float32,
-                over_shape_values, [[4, 4]],
+                over_shape_values, [[8, 4, 4]],
                 use_system_shared_memory=TEST_SYSTEM_SHARED_MEMORY,
-                use_cuda_shared_memory=TEST_CUDA_SHARED_MEMORY)
-        except InferenceServerException as ex:
-            self.assertEqual("inference:0", ex.server_id())
+                use_cuda_shared_memory=TEST_CUDA_SHARED_MEMORY,
+                batch_size=8)
+        # InferenceServerException will be raised from different namespace,
```

Comment:
Do you mean because InferenceServerException comes from grpcclient.utils? @tanmayv25 it is confusing that there is an identical utils.py in both httpclient.utils and grpcclient.utils. Especially when you are trying to use both in the same file. I think you asked this before and I misunderstood but it probably makes sense to just have a tritonclient.utils model that both can depend on. What do you think?

Reply:
Depending on the client, the exception will be either `grpcclient.utils.InferenceServerException` or `httpclient.utils.InferenceServerException`, so here uses their comment base class to avoid catching them separately.

I agree, and to make everything into one package, we may have `tritonclient` and have three components: `utils`, `grpc`, `http`. User can choose to build with both or one of grpc and http.